src/codeu/chat/server/View.java

```diff
@@ -46,15 +46,15 @@ public View(Model model) {
 
   @Override
   public Collection<User> getUsers(Collection<Uuid> ids) {
-    return model.userById(intersect(ids, false), "ASC");
+    return model.userById(ids, false, true);
```

Comment:
I would argue that this is even more of a reason to remove it. If you don't know whether a piece of code will be used, then don't include it. Andre can always add what he needs when he makes his changes. The general rule to go by here is to write the minimal amount of code to get the job done in a way that is easy to understand for other engineers. Code that goes unused always raises questions in developers minds and tends to never get cleaned up because people don't know whether it is used or not.